### DISCUSSION TITLE
Welcome to Threats Manager Discussions!

### DISCUSSION THREAD

**Discussion by simonec73** [simonec73/threatsmanager#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * If you want to report an issue, do not do that here, but use the [Issues page](https://github.com/simonec73/threatsmanager/issues), please.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jgadsden** [comment]
Hello @simonec73 , it is good to see Threats Manager taking shape; nice to have various tools when it comes to threat modeling. This project seems to complement Threat Dragon (I am one of this project's leaders) - they are very different projects but with the same goal of providing easy to use tools that are free to use

**Reply by simonec73** [reply]
Thank you, @jgadsden! 
I do not see that much difference between our products. On the contrary, I look forward collaborating and possibly merge the two initiatives. My intent is to provide not only a desktop application, but also a re-hostable engine that can be used for instance in Threat Dragon. What do you think about this idea? Let's talk.

**Reply by jgadsden** [reply]
Hello @simonec73 , yes, certainly a threat engine would be great to collaborate on. The Threat Dragon engine is certainly minimalist :-) at the moment and certainly this is something the Threat Dragon leaders (myself, @lreading and @mike-goodwin ) have long felt should be looked at

The big thing with Threat Dragon is that it is entirely open source and cross platform. So we would have to retain that in whatever the two projects do together

**Reply by simonec73** [reply]
Open Source and cross-platform. Yes, I confirm that Threats Manager Platform is both as well. In fact, it is currently based on .NET Core, and I am planning to invest some time to support .NET 5 or even .NET Standard. I expect gaps, but I'm sure they can be addressed.

**Reply by jgadsden** [reply]
That sounds great to me, thanks @simonec73 . I have added you as a maintainer of the Threat Dragon project, you should have  received an invite, hope that is OK with you

I am not sure if that is how you want to proceed with this, there are various options:
* either create an npm package for the threatsmanager which Threat Dragon has as a dependency (not sure if this is possible)
* or add this to Threat Dragon directly in the code (again not sure if there is a language barrier)
* or some other way I can not think of right now :-)

@lreading is migrating Threat Dragon to Vue on a dedicated branch, with this version 2.0 going live sometime later this year (originally estimated September 2021, but he is powering through it so it may be earlier). @simonec73 you may want threatsmanager maybe in version 2.0 or earlier / later - over to you on when really
